selfdrive/thermald/fan_controller.py

```diff
@@ -92,7 +92,7 @@ def update(self, max_cpu_temp: float, ignition: bool) -> int:
     if ignition != self.last_ignition:
       self.controller.reset()
 
-    error = 75 - max_cpu_temp
+    error = (70 if ignition else 75) - max_cpu_temp
```

Comment:
It might also be fine to lower the offroad setpoint to 70C as well. This is the offroad distribution we get now (so only a small percentage of devices is >70C), and once we lower power consumption it should be even better.
![image](https://user-images.githubusercontent.com/894159/163996008-df38583b-3132-4d02-8d25-28d1cd9f104f.png)